elements/bulbs-slideshow/bulbs-slideshow.js

```diff
@@ -103,9 +109,18 @@ class BulbsSlideshow extends BulbsHTMLElement {
 
   afterSlideTransition () {
     this.render();
+    this.sendPageView.bind(this).call();
```

Comment:
`sendPageView` doesn't have access to `analyticsManager` or `analyticsWrapper`, etc IIRC